src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+html {
+  --activeted: url(/src/images/star-active.svg);
+  --inactivated: url(/src/images/star.svg);
```

Comment:
You could also declare css vars at :root, but there's nothing wrong with it, just a better convention